rest-server/src/config/token.js

```diff
@@ -15,19 +15,17 @@
 // DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
-
 // module dependencies
 const Joi = require('joi');
 const jwt = require('express-jwt');
 const config = require('./index');
 
-
 const jwtCheck = jwt({
   secret: config.jwtSecret
 });
 
-// define auth schema
-const authSchema = Joi.object().keys({
+// define input schema
+const inputSchema = Joi.object().keys({
```

Comment:
this is not the only input schema, use the model name instead of `input`

Reply:
The context is clear. No need to mention the module name again.